NStackSDK/NStackSDK/Classes/Translations/TranslationManager.swift

```diff
@@ -36,7 +36,7 @@ public class TranslationManager {
     var translationsObject: Translatable?
     
     /// In memory cache of the last language object.
-    public fileprivate(set) var currentLanguage: Language?
+    public var currentLanguage: Language?
```

Comment:
Should this be changeable outside the framework? I think this property should stay public read-only with internal setter. For the language overriding feature users can use the `languageOverride` property.